blocks/library/paragraph/index.js

```diff
@@ -248,6 +249,21 @@ registerBlockType( 'core/paragraph', {
 				),
 			},
 		],
+		to: [ 'left', 'center', 'right' ].map( ( value ) => ( {
+			type: 'shortcut',
+			shortcut: value.charAt( 0 ),
+			transform( attributes ) {
+				const firstAlign = first( attributes ).align;
+				const isSame = attributes.every( ( { align } ) => align === firstAlign );
```

Comment:
My unitiated perspective: What is the shape of `attributes` ? I would have expected an object for a single block, but this looks to be an array. Are these an array for blocks in a multi-selection? A bit unfortunate overloading of the term `attributes` between an object of a single block, and an array of multiple blocks.

Reply:
Yeah, it's similar in other transforms. Elsewhere it has been named `blockAttributes`, which doesn't really feel any better... Any ideas?

Reply:
Ah, I rambled on about the same thing in a different hunk. :) See: